src/test/java/uk/gov/hmcts/reform/jobscheduler/FixtureData.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.jobscheduler;
+
+import feign.FeignException;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.authorisation.exceptions.AbstractAuthorisationException;
+
+import java.util.Collections;
+
+public final class FixtureData {
+
+    private FixtureData() {
+        // empty constructor
+    }
+
+    public static AbstractAuthorisationException setUpUnauthorisedAccess() {
```

Comment:
The method name kind of reads weird (especially when used: _'given XXX will Throw setUpUnauthorisedAccess'_.
It's not taking any parameters, nor is it modifying any state - it's creating an exception.

Perhaps `getAuthorisationException`?